content/en/docs/collector/Install/binary/linux.md

```diff
@@ -0,0 +1,140 @@
+---
```

Comment:
We'll need to add a landing page (`index.md`) to the `binary` directory. It should be called "Install from a Collector binary" and say "You can install the OpenTelemetry Collector by downloading and running a prebuilt binary. Choose your operating system below for installation steps."